include/index_helper.cuh

```diff
@@ -165,6 +178,20 @@ namespace quda {
   __device__ __host__ inline int linkIndexP1(const Coord &x, const I X[4], const int mu) {
     return linkIndexDn<I, 1>(x, X, mu);
   }
+  
+ /**
+     Compute the checkerboard 1-d index from the 4-d coordinate x[] +2 in the mu direction
+
+     @return 1-d checkerboard index
```

Comment:
Missing [in] / [out] labels from the parameters, and the @return should be after the parameter list.

Reply:
fixed (also other places in the file)